neqo-http3/src/frames/hframe.rs

```diff
@@ -142,7 +142,9 @@ impl HFrame {
                 element_id,
                 priority,
             } => {
-                let mut update_frame = Encoder::new();
+                let mut out = vec![];
+                // TODO: separate encoder needed?
+                let mut update_frame = Encoder::new(&mut out);
```

Comment:
Anything left to be done about this `TODO`?

Reply:
Cleaned up in https://github.com/mozilla/neqo/pull/2093/commits/ffc3708dbd9426946caa90bd0183c17d38515a79.